data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -2385,6 +2385,101 @@
 },
 {
 "type" : "recipe",
+"result": "chem_hexamine",
+"category": "CC_CHEM",
+"subcategory": "CSC_CHEM_CHEMICALS",
+"skill_used": "cooking",
+"difficulty": 6,
+"time": 30000,
+"batch_time_factors": [ 67, 5 ],
+"book_learn": [[ "recipe_labchem", 6 ]],
+"qualities":[
+{"id":"BOIL","level":2},
+{"id":"CHEM","level":3}
+],
+"components": [
+[
+[ "ammonia", 4 ]
+],
+[
+[ "chem_formaldehyde", 6 ]
+]
+]
+},
+{
+"type" : "recipe",
+"result": "chem_formaldehyde",
+"category": "CC_CHEM",
+"subcategory": "CSC_CHEM_CHEMICALS",
+"skill_used": "cooking",
+"difficulty": 5,
+"time": 15000,
+"batch_time_factors": [ 80, 4 ],
+"book_learn": [[ "textbook_chemistry", 5 ], [ "adv_chemistry", 5 ]],
+"qualities":[
+{"id":"BOIL","level":2},
+{"id":"CHEM","level":2}
+], "tools": [
+[ [ "surface_heat", 50, "LIST" ] ]
+],
+"components": [
+[
+[ "chem_methanol", 50 ]
+]
+]
+},
+{
+"type" : "recipe",
+"result": "chem_methanol",
+"byproducts": [ [ "chem_ethanol", 2 ] ],
+"category": "CC_CHEM",
+"subcategory": "CSC_CHEM_CHEMICALS",
+"skill_used": "cooking",
+"difficulty": 4,
+"time": 10000,
+"batch_time_factors": [ 80, 4 ],
+"autolearn": true,
+"qualities":[
+{"id":"BOIL","level":2},
+{"id":"DISTILL","level":1}
+], "tools": [
+[ [ "water_boiling_heat", 25, "LIST" ] ]
+],
+"components": [
+[
+[ "denat_alcohol", 1000 ]
+]
+]
+},
+{
+"type" : "recipe",
+"result": "chem_methanol",
+"id_suffix": "from wood",
+"category": "CC_CHEM",
+"subcategory": "CSC_CHEM_CHEMICALS",
+"skill_used": "cooking",
+"difficulty": 4,
+"time": 60000,
+"batch_time_factors": [ 80, 4 ],
+"autolearn": true,
+"qualities":[
+{"id":"BOIL","level":2},
+{"id":"DISTILL","level":1}
+], "tools": [
+[ [ "surface_heat", 25, "LIST" ] ]
+],
+"components": [
+[
+[ "splinter", 40 ]
```

Comment:
A splinter is 1lb so this is 40lb of wood. Maybe allow 12 two by fours as an alternative ingred?

Reply:
Sure.

I chose that number very arbitrarily. Let me know if it should be changed.

Reply:
Yep, that yields 250mL.